### DISCUSSION TITLE
Separate rtsp and live streaming

### DISCUSSION THREAD

**Discussion by dam034** [iamscottxu/obs-rtspserver#78] [Ideas]
Hi,

I want to propose a change to the wonderful rtspserver plugin.

I would like the rtsp server to be independent of the live stream that OBS can do. I'll explain.

In settings, if possible I want another tab where I can configure the video quality, bitrate, etc, as recording and streaming:
![rtsp2](https://user-images.githubusercontent.com/87028345/128374748-54db9142-46de-4bd2-b395-80ac1b806500.png)

In the main window I want another independent button to manage the server:
![rtsp1](https://user-images.githubusercontent.com/87028345/128375454-94acad2c-0174-47d7-80d8-cfcaf897626b.png)

I am proposing this to resolve the fact that if the RTSP server window is open, as shown here:
![rtsp3](https://user-images.githubusercontent.com/87028345/128375655-3c2340d5-9de9-4ece-b883-390816a4e13f.png)
I cannot go back to the main screen to manage the outgoing scene

I hope I was clear. If necessary, I am willing to provide further details.

**Comment by iamscottxu** [comment]
Your idea is good, but because of the limitations of OBS's plug-in API, the above is difficult to achieve.
